src/test/java/org/broadinstitute/hellbender/utils/codecs/gtf/GencodeGtfCodecUnitTest.java

```diff
@@ -3495,6 +3498,46 @@ private GencodeGtfGeneFeature createGencodeGtfGene_gencode_v19_and_this_is_a_val
                                  "##date: 2014-07-25" ),
                                  true },                                 // Good Header!
 
+                // -------------
+
+                { Arrays.asList( "#!genome-build ASM584v2",
+                        "#!genome-version ASM584v2",
+                        "#!genome-date 2014-08",
```

Comment:
similar to above comment, add a too short header.

Reply:
Sure.  Fixed.